client.go

```diff
@@ -309,6 +309,29 @@ func (c *Client) Version(ctx context.Context) (string, error) {
 	return response.Version, nil
 }
 
+// Status returns the current state of the KES server.
+func (c *Client) Status(ctx context.Context) (State, error) {
```

Comment:
```suggestion
func (c *Client) Status(ctx context.Context) (*State, error) {
```

It would be better to return `*State` instead of `State`? so return will be: `return nil, err` instead of returning an empty structure